src/Plugin/PreviewLinkAutopopulate/Publications.php

```diff
@@ -100,7 +100,7 @@ public function getPreviewEntities(): array {
           'localgov_publication' => array_keys($book_links),
         ]);
         foreach ($cover_pages as $cover_page) {
-          if ($cover_page instanceof NodeInterface && !isset($nodes[$cover_page->id()])) {
+          if (!isset($nodes[$cover_page->id()])) {
```

Comment:
It will be a NodeInterface, though the analysis won't know that I think. Interested to know why it wanted that one removed.

Either was sure it's fine.